lib/led-matrix.cc

```diff
@@ -159,7 +161,7 @@ bool RGBMatrix::SetPWMBits(uint8_t value) {
   }
   return success;
 }
-uint8_t RGBMatrix::pwmbits() { return active_->framebuffer()->pwmbits(); }
+uint8_t RGBMatrix::pwmbits() { return pwm_bits_; }
```

Comment:
Why did you change this ? (also for luminance correct).
The contract is, that these return the value for the currently active buffer.